src/utils/getLocaleStorage.js

```diff
@@ -0,0 +1,13 @@
+export const getLocaleStorage = (key) => {
+  const value = window.localStorage.getItem(key);
+
+  if (value) {
+    try {
+      return JSON.parse(value);
+    } catch {
+      return null;
```

Comment:
It's good that you're catching exceptions when parsing JSON, but it would be helpful to log the error message to the console for debugging purposes. You can do this by catching the error in a variable (e.g., `catch (e)`), and then logging it (`console.error(e)`).